### PR TITLE
Example updates for s3

### PR DESCRIPTION
**Pull request by elysahall**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: awscli/examples/s3/sync.rst [hunk 1]
```diff
@@ -1,112 +1,119 @@
...
-the size of the s3 object, the last modified time of the local file is newer than the last modified time of the s3
+uploading the local files to S3.  A local file will require uploading if the size of the local file is different than
+the size of the S3 object, the last modified time of the local file is newer than the last modified time of the S3
 object, or the local file does not exist under the specified bucket and prefix.  In this example, the user syncs the
 bucket ``mybucket`` to the local current directory.  The local current directory contains the files ``test.txt`` and
-``test2.txt``.  The bucket ``mybucket`` contains no objects::
+``test2.txt``.  The bucket ``mybucket`` contains no objects. ::
 
-    aws s3 sync . s3://mybucket
+    aws s3 sync . S3://mybucket
```
Comment (kdaily) [inline awscli/examples/s3/sync.rst:10]: This should still be lowercase.

File: awscli/examples/s3/sync.rst [hunk 2]
```diff
@@ -1,112 +1,119 @@
...
 
 Output::
 
-    upload: test.txt to s3://mybucket/test.txt
-    upload: test2.txt to s3://mybucket/test2.txt
+    upload: test.txt to S3://mybucket/test.txt
```
Comment (kdaily) [inline awscli/examples/s3/sync.rst:14]: Lowercase

File: awscli/examples/s3/sync.rst [hunk 3]
```diff
@@ -1,112 +1,119 @@
...
+    upload: test2.txt to S3://mybucket/test2.txt
```
Comment (kdaily) [inline awscli/examples/s3/sync.rst:15]: Lowercase

File: awscli/examples/s3/sync.rst [hunk 4]
```diff
@@ -1,112 +1,119 @@
...
-the bucket ``mybucket2``.  The bucket ``mybucket`` contains the objects ``test.txt`` and ``test2.txt``.  The bucket
+prefix and bucket by copying S3 objects. An S3 object will require copying if the sizes of the two S3 objects differ,
+the last modified time of the source is newer than the last modified time of the destination, or the S3 object does not
+exist under the specified bucket and prefix destination. 
+
+In this example, the user syncs the bucket ``mybucket`` to the bucket ``mybucket2``. The bucket ``mybucket`` contains the objects ``test.txt`` and ``test2.txt``. The bucket
 ``mybucket2`` contains no objects::
 
-    aws s3 sync s3://mybucket s3://mybucket2
+    aws s3 sync S3://mybucket S3://mybucket2
```
Comment (kdaily) [inline awscli/examples/s3/sync.rst:27]: Lowercase

**Review by kdaily — CHANGES_REQUESTED** [review]
Looks good overall, only changes I saw are in:

awscli/examples/s3/sync.rst

Looks like a find/replace was used and put uppercase `S3` inside commands for the S3 uri, which should be lowercase (`s3://`).

**Review by kdaily — APPROVED** [review]

**Review by kyleknap — APPROVED** [review]

**Pull request merged**